libs/brig-types/src/Brig/Types/TURN.hs

```diff
@@ -96,8 +96,8 @@ data Scheme = SchemeTurn
             | SchemeTurns
             deriving (Eq, Show, Generic)
 
-data Transport = TransportTCP
-               | TransportUDP
+data Transport = TransportUDP
```

Comment:
Since the order is apparently important, I'd appreciate a comment here (maybe even a link to the relevant issue).

Reply:
It _was_ important, not anymore 😊 See https://github.com/wireapp/wire-server/pull/273/commits/f7fa0877c83f9f538c06c04ae610d05b7c209ea6 and let me know if it's clearer ?

Reply:
> ```diff haskell
> +-- | Relies on Ord instances for `Scheme` and `Transport`.
> +-- | First, prefer `turn` scheme and then prefer UDP.
> ```

It looks like it wasn't important and now it is important. So, still not clear.